tasks/libs/pipeline/github_slack_review_map.yaml

```diff
@@ -7,7 +7,7 @@
 '@datadog/agent-all': '#datadog-agent-pipelines'
 '@datadog/agent-apm': '#apm-agent'
 '@datadog/agent-configuration': '#agent-configuration'
-'@datadog/agent-cspm': '#k9-cspm'
+'@datadog/agent-cspm': '#k9-cspm-ops'
```

Comment:
I think #security-and-compliance-agent is probably better, k9-cspm-ops is mostly backend stuff